ipynb/examples/android/workloads/Android_Viewer.ipynb

```diff
@@ -0,0 +1,312 @@
+    "    # Not required if there is only one device connected to your computer\n",
+    "    \"device\"       : \"0123456789ABCDEF\",\n",
+    "    \n",
+    "    # Android home\n",
+    "    # Not required if already exported in your .bashrc\n",
+    "    \"ANDROID_HOME\" : \"/home/vagrant/lisa/tools/\",\n",
+    "\n",
+    "    # Folder where all the results will be collected\n",
+    "    \"results_dir\" : \"Viewer_example\",\n",
+    "\n",
+    "    # Define devlib modules to load\n",
+    "    \"modules\"     : [\n",
+    "        'cpufreq'       # enable CPUFreq support\n",
+    "    ],\n",
+    "\n",
+    "    # FTrace events to collect for all the tests configuration which have\n",
+    "    # the \"ftrace\" flag enabled\n",
+    "    \"ftrace\"  : {\n",
+    "         \"events\" : [\n",
+    "            \"sched_switch\",\n",
+    "            \"sched_wakeup\",\n",
+    "            \"sched_wakeup_new\",\n",
+    "            \"sched_overutilized\",\n",
+    "            \"sched_load_avg_cpu\",\n",
+    "            \"sched_load_avg_task\",\n",
+    "            \"sched_load_waking_task\",\n",
+    "            \"cpu_capacity\",\n",
+    "            \"cpu_frequency\",\n",
+    "            \"cpu_idle\",\n",
+    "            \"sched_tune_config\",\n",
+    "            \"sched_tune_tasks_update\",\n",
+    "            \"sched_tune_boostgroup_update\",\n",
+    "            \"sched_tune_filter\",\n",
+    "            \"sched_boost_cpu\",\n",
+    "            \"sched_boost_task\",\n",
+    "            \"sched_energy_diff\"\n",
+    "         ],\n",
+    "         \"buffsize\" : 100 * 1024,\n",
+    "    },\n",
+    "\n",
+    "    # Tools required by the experiments\n",
+    "    \"tools\"   : [ 'trace-cmd', 'taskset'],\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "deletable": true,
+    "editable": true
+   },
+   "outputs": [],
+   "source": [
+    "# Initialize a test environment using:\n",
+    "te = TestEnv(my_conf, wipe=False)\n",
+    "target = te.target"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "deletable": true,
+    "editable": true
+   },
+   "source": [
+    "# Workload definition\n",
+    "\n",
+    "The Viewer workload will simply read an URI and let Android pick the best application to view the item designated by that URI. That item could be a web page, a photo, a pdf, etc. For instance, if given an URL to a Google Maps location, the Google Maps application will be opened at that location. If the device doesn't have Google Play Services (e.g. HiKey960), it will open Google Maps through the default web browser.\n",
+    "\n",
+    "The Viewer workload expects a \"workload\" argument that is a function pointer to some test code to execute while the item is being displayed. In this case we'll simply execute a script on the target, see **${LISA_HOME}/libs/util/target_script.py**"
+   ]
```

Comment:
Can you add a brief explanation of what the `TargetScript` biz is doing? It might be confusing to a new reader.